Fabric.Authorization.API/RemoteServices/Identity/Providers/IdentityServiceProvider.cs

```diff
@@ -47,23 +49,21 @@ public IdentityServiceProvider(IHttpClientFactory httpClientFactory, IAppConfigu
             var tokenClient = new TokenClient(tokenUriAddress, "fabric-authorization-client", settings.ClientSecret);
             var accessTokenResponse = await tokenClient.RequestClientCredentialsAsync(IdentityScopes.SearchUsersScope).ConfigureAwait(false);
 
-            var httpClient = new HttpClientFactory(
-                tokenUriAddress,
-                "fabric-authorization-client",
-                settings.ClientSecret,
-                null,
-                null).CreateWithAccessToken(new Uri(settings.Authority), accessTokenResponse.AccessToken);
+            var httpRequestMessage = _httpRequestMessageFactory.CreateWithAccessToken(HttpMethod.Post, new Uri($"{settings.Authority}api/users"),
```

Comment:
Should we ensure that there is a trailing slash on the Authority, otherwise we could get a bad uri and it would be a hard bug to decipher.